src/app/messages/messages.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { MessageService } from '../message.service';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-messages',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './messages.component.html',
+  styleUrl: './messages.component.css'
+})
+export class MessagesComponent {
+
+  constructor(public messageService: MessageService) {}
```

Comment:
instance service is public to acceess it's functions in the component template 👍